Application/src/visuals/Trail.java

```diff
@@ -0,0 +1,50 @@
+package visuals;
+
+/**
+ * Class for the trail of the message
```

Comment:
Don't make the header dependable on another class. Just say what this class does, not what it's currently implemented in.Basically make it more generic, and remove mention of message.